package.json

```diff
@@ -83,14 +86,15 @@
     "normalize.css": "^5.0.0",
     "postcss-loader": "^1.0.0",
     "precss": "^1.4.0",
+    "prop-types": "^15.5.8",
     "react": "^15.3.2",
     "react-dom": "^15.3.2",
     "react-dropdown": "^1.1.0",
     "react-hot-loader": "3.0.0-beta.6",
     "react-mdl": "^1.7.2",
-    "react-relay": "^0.9.3",
+    "react-relay": "dev",
     "react-router": "^2.8.1",
-    "react-router-relay": "^0.13.5",
+    "react-router-relay": "https://github.com/Neitsch/react-router-relay#fullstack",
```

Comment:
I'm kind of hesitant to merge this in, but since it is canary, I guess it is okay...